Wrappers/Python/cil/processors/Masker.py

```diff
@@ -18,23 +18,52 @@
 # CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
 
 from cil.framework import DataProcessor, AcquisitionData, ImageData, ImageGeometry, DataContainer
-import warnings
 import numpy
 from scipy import interpolate
 
 class Masker(DataProcessor):
     r'''
-    Processor to fill missing values provided by mask. Please use the desiried method to configure a processor for your needs.
+    Processor to fill missing values provided by mask.
+    Please use the desired method to configure a processor for your needs.
+
+    Parameters
+    ----------
+    mask : DataContainer, ImageData, AcquisitionData, numpy.ndarray
+        A boolean array with the same dimensions as input, where 'False' represents masked values.
+        Alternatively an integer array where 0 represents masked values, and any other value represents unmasked values.
+        Mask can be generated using 'MaskGenerator' processor to identify outliers. 
+    mode : {'value', 'mean', 'median', 'interpolate'}, default='value'
+        The method to fill in missing values 
+    value : float, default=0
+        Substitute all outliers with a specific value if method='value', otherwise discarded.
+    axis : str or int
+        Specify axis as int or from 'dimension_labels' to calculate mean, median or interpolation
+        (depending on mode) along that axis
+    method : {'linear', 'nearest', 'zeros', 'linear', 'quadratic', 'cubic', 'previous', 'next'}, default='linear'
+        Interpolation method to use.
+    
+    Returns
+    -------
+    DataContainer or its subclass with masked outliers
+
```

Comment:
```suggestion
```

Reply:
what's wrong with the returns section?

Reply:
is it because it doesn't return anything unless it's run

Reply:
This is the docstring of the class; it refers really to `__init__` which never returns.